src/drawtypes/label.cpp

```diff
@@ -115,12 +127,33 @@ namespace drawtypes {
 
     string text;
 
+    struct sides padding, margin;
+
     if (required) {
       text = conf.get<string>(section, name);
     } else {
       text = conf.get<string>(section, name, move(def));
     }
 
+    const auto get_left_right = [&](string key) {
+      int left = conf.get<int>(section, key + "-left", 0),
+          right = conf.get<int>(section, key + "-right", 0),
+          all = conf.get<int>(section, key, 0);
```

Comment:
Use the `{padding,margin}` parameter as main and `-{left,right}` as override, i.e:

~~~ cpp
auto value = conf.get<int>(section, key, 0);
auto left = conf.get<int>(section, key + "-left", value);
auto right = conf.get<int>(section, key + "-right", value);
return {left, right};
~~~